docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+*******************************************************
+Storage Quota at /home/
+*******************************************************
+Staying under your ``/home/<username>`` quota is vital in preventing issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
```

Comment:
I don't want to reference the storage quota other than pointing to the page that has it because if it changes in the future, we only have to change it in one location and not look through the RTD and find the old references. I could be swayed on this but I figure this would be easier for future references.